Week3/homework/step2-3.js

```diff
@@ -4,40 +4,57 @@
 function repeatStringNumTimesWithFor(str, num) {
   // eslint-disable-next-line prefer-const
   let result = '';
-
-  // Replace this comment and the next line with your code
-  console.log(str, num, result);
-
+  for (let i = 0; i < num; i++) {
+    if (num < 0) {
+      result += '';
+    } else {
+      result += str;
+    }
+  }
   return result;
 }
 
 console.log('for', repeatStringNumTimesWithFor('abc', 3));
+console.log('for', repeatStringNumTimesWithFor('abc', -2));
 
 // Use a 'while' loop
 function repeatStringNumTimesWithWhile(str, num) {
   // eslint-disable-next-line prefer-const
   let result = '';
-
-  // Replace this comment and the next line with your code
-  console.log(str, num, result);
-
+  let i = 0;
+  while (i < num) {
+    if (num < 0) {
```

Comment:
Same as `for` loop. I think the `if` condition needs to be outside the `loop`. 🙂